rasa/core/tracker_store.py

```diff
@@ -285,10 +249,12 @@ def deserialise_tracker(
 
         try:
             dialogue = Dialogue.from_parameters(json.loads(serialised_tracker))
-        except UnicodeDecodeError:
-            dialogue = self._deserialize_dialogue_from_pickle(
-                sender_id, serialised_tracker
-            )
+        except UnicodeDecodeError as e:
+            raise ValueError(
```

Comment:
Should we use a `RasaException` to avoid that custom trackers raise this?

Reply:
I was umming and ahing about this... I chose not to use `RasaException` because I didn't want us to potentially miss a bug we introduce in the tracker serialisation. But on second thought, we have tests and this is more likely to caused by some custom tracker store - especially `UnicodeDecodeError` which most likely means trying to `json.load` bytes.